src/routes/pig.py

```
@@ -136,12 +136,12 @@ async def join_pig(id: int, session: SessionDep, request: Request):
 
 
 @pig_router.post('/pig/{id}/member/leave', status_code=204)
-async def leave_pig(id: int, session: SessionDep, scsc_global_status: SCSCGlobalStatusDep, request: Request):
+async def leave_pig(id: int, session: SessionDep, request: Request):
     current_user = get_user(request)
     pig = session.get(PIG, id)
     if not pig: raise HTTPException(404, detail="해당 id의 시그/피그가 없습니다")
     if pig.owner == current_user.id: raise HTTPException(409, detail="시그/피그장은 해당 시그/피그를 탈퇴할 수 없습니다")
-    pig_member = session.exec(select(PIGMember).where(PIGMember.ig_id == id).where(PIGMember.user_id == current_user.id).where(PIGMember.status == scsc_global_status.status)).first()
+    pig_member = session.exec(select(PIGMember).where(PIGMember.ig_id == id).where(PIGMember.user_id == current_user.id)).first()
     if not pig_member: raise HTTPException(404, detail="시그/피그의 구성원이 아닙니다")
```

Ruff (0.12.2)

Multiple statements on one line (colon)

(E701)